src/scss/tom-select.bootstrap5.scss

```
@@ -205,6 +205,16 @@ $select-arrow-offset: calc(#{$select-padding-x} + 5px) !default;
 		}
 	}
 
+	&.single {
+        &.has-items.input-hidden {
```

```suggestion
        &.has-items {
```

Removing `.input-hidden` makes this fix also work when you use the Dropdown Input plugin.